packages/core/contracts/financial-templates/perpetual-multiparty/PerpetualPositionManager.sol

```diff
@@ -576,13 +595,18 @@ contract PerpetualPositionManager is FeePayer, AdministrateeInterface {
     /****************************************
      *        GLOBAL STATE FUNCTIONS        *
      ****************************************/
+
+    function applyFundingRate() external notEmergencyShutdown() updateFundingRate() nonReentrant() {
```

Comment:
Smart to make this `notEmergencyShutdown()` if we're not applying `updateFundingRate()` on `settleEmergencyShutdown` +1